besu/terraform/aws/ibftScale/ibft4caliper/variables.tf

```diff
@@ -100,10 +96,23 @@ variable "bootnode_instance_type" {
   default = "c5d.4xlarge"
 }
 
-variable "rpcnode_instance_type" {
-  default = "t3.medium"
```

Comment:
Is this size appropriate for maximum throughput? Generally I test with all nodes as `c5d.4xlarge`, as it's the instance type that seems to provide the greatest performance. It made sense to do it this way, as we were searching for software-limited bottlenecks, but I don't know if it makes sense to provision Orion that way or not.